Okta.AspNet.Abstractions/Okta.AspNet.Abstractions.csproj

```diff
@@ -16,14 +16,24 @@
 
   <ItemGroup>
     <PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="5.4.0" />
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.0-beta008" PrivateAssets="All" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.4.0" />
     <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
     <AdditionalFiles Include="..\stylecop.json" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="5.4.0" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.4.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.0' ">
+    <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="5.5.0" />
```

Comment:
The latest version of [`Microsoft.IdentityModel.Tokens`](https://www.fuget.org/packages/Microsoft.IdentityModel.Tokens/5.6.0) is `5.6.0` and it is compatible with .NetStandard 2.0 (as previous versions). Why did you use `5.4.0` for `netstandard2.0` and `5.5.0` for `netcoreapp3.0` since they both have the same compatibility?

Reply:
I got the libraries wrong in the PR description (have updated). `Microsoft.AspNetCore.Authentication.JwtBearer` & `Microsoft.AspNetCore.Authentication.OpenIdConnect` need to be upgraded from 2.2 to 3.0 and `Microsoft.IdentityModel.Tokens` is a just dependency.

Reply:
Do you have an ETA for 3.0 support?

Reply:
We are now looking into it, and planning to release a major version soon. We don't have a specific release date to share right now, but we expect to have it available as soon as we can.